src/classifier/Pipe.h

```diff
@@ -29,6 +29,11 @@
 #include "Parameters.h"
 #include "AlgUtils.h"
 
+
+#include <thread>         // std::thread
```

Comment:
No need for comments here, and if you need them, they should be a single space apart.

Not sure if this will compile in Linux right now...